domain/src/ValueObjects/Transaction.php

```diff
@@ -29,8 +29,10 @@ public function __construct(
         public readonly bool $receivedAssetIsNft,
         public readonly ?string $transactionFeeCurrency,
         public readonly Quantity $transactionFeeQuantity,
+        public readonly ?FiatAmount $transactionFeeCostBasis,
         public readonly ?string $exchangeFeeCurrency,
         public readonly Quantity $exchangeFeeQuantity,
+        public readonly ?FiatAmount $exchangeFeeCostBasis,
```

Comment:
ℹ️ These properties are required in case the fees are not denominated in pounds, which they often won't be